src/day_1/day_1.js

```diff
@@ -1,4 +1,22 @@
 class Measurement {
+  group(data) {
+    try {
+      let arrs = [],
+        size = 3;
```

Comment:
Would be nice if the size is passed by parameters to the method.

Reply:
Modified.